crates/extensions/c8y_mapper_ext/src/tests.rs

```diff
@@ -2547,7 +2548,7 @@ pub(crate) async fn spawn_c8y_mapper_actor(
     let device_name = "test-device".into();
     let device_topic_id = EntityTopicId::default_main_device();
     let device_type = "test-device-type".into();
-    let service_type = "service".into();
+    let config = TEdgeConfigRepository::load_toml_str("service.ty = \"service\"");
```

Comment:
Ideally we would be able to parse a fake `tedge.toml` string for all of the values here, but we wouldn't be able to control the device ID without generating a certificate. Currently, the need to parse toml is due to use of a tedge config struct to simplify a function signature (see comment in crates/extensions/tedge_health_ext/src/lib.rs for more details). Finding a way to mitigate this would be nice. In general, I would prefer if we could use some mechanism like this more generally as it would be testing the public API, but obviously right now is not the time to deal with such a concern.

Reply:
>  we wouldn't be able to control the device ID without generating a certificate.

This is also a pain for the users as `tedge config get device.id` is currently meaningful only on the device where the certificate is stored.